app/styles/partials/nav-bar.scss

```diff
@@ -18,3 +18,16 @@
 .blue.text {
   color: #2185d0 !important;
 }
+
+.item.ui.dropdown {
+  min-width: 200px;
+}
+
+.min-width-200 {
```

Comment:
Please name it appropriately. `min-width-200` is not a correct name. Since this anyway going to be used _Only_ for the user dropdown in the navbar.